Makefile

```diff
@@ -35,7 +35,7 @@ test-legacy: ## Runs test suite with output from tests printed
 	@cargo test -- --nocapture -Zunstable-options --report-time
 
 test: build $(EF_TESTS_DIR) ## Runs test suite using next test
-	@cargo nextest run --workspace --all-features --no-fail-fast $(filter-out $@,$(MAKECMDGOALS))
+	RISC0_DEV_MODE=1 cargo nextest run --workspace --all-features --no-fail-fast $(filter-out $@,$(MAKECMDGOALS))
```

Comment:
RISC0_DEV_MODE=1 - is it okay? It will not produce proofs so `make test` will fail on basic_prover_test

Reply:
no dev mode produces "fake proofs" which are verifiable by processes that also have risc0_dev_mode=1 :)